packages/tabs/src/components/tabswidget.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+	<div>
+		<div
+			ref="tablist"
+			class="t-ui-tabswidget"
+			role="tablist"
+			@click="clickHandler"
+			@keydown="keydownHandler"
+		>
+			<slot name="tabs"></slot>
```

Comment:
I wonder if we could simplify the implementation by using a prop instead of a slot here. Would probably also reduce the amount of code we would have to write in the event handlers down here. At the moment it seems like we miss to leverage most of the benefits Vue provides us for modifying attributes.

Reply:
Reasoning behind using `<tournant-tab>` was that that way, people could place more than just text in Tabs, for example an icon. This wouldn't be possible if tabs were an array, e.g. `["Foo", "Bar", "Baz"]`. I know what you mean regarding this approach not being vue-ish, though. I'm wondering for quite some while now: is there a third way?

Reply:
Hmm … good point. Let me think about this. Of the top of my head: When using an array of objects we could provide some level of customisation (e.g. class names and text). If we change `Tab.vue` into a functional component we might(!) be able to provide an option to render child components through it (say you want to add an icon that is a component). Either way, we should be very clear what is and what is not usable (e.g. don’t put complex structures in there as the button will likely eat its semantics). Locking this down has some advantages, as keeping it open. 

If we keep TournantTab, we have to make sure that the name for the ID is slugified. A random ID wouldn’t work here as Tab and Tabpanel need to have the same name, right? 

In the end: It depends. Need to get those stickers now.

Reply:
> If we keep TournantTab, we have to make sure that the name for the ID is slugified. A random ID wouldn’t work here as Tab and Tabpanel need to have the same name, right?

Correct.